pkg/sso/client_test.go

```diff
@@ -50,6 +59,22 @@ func TestClientAuthorizeURL(t *testing.T) {
 	}
 }
 
+func TestClientAuthorizeURLWithNoDomainAndProvider(t *testing.T) {
+	client := Client{
+		APIKey:      "test",
+		ProjectID:   "proj_123",
+		RedirectURI: "https://example.com/sso/workos/callback",
+	}
+
+	u, err := client.GetAuthorizationURL(GetAuthorizationURLOptions{
+		State: "state",
+	})
+
+	var expectedURL *url.URL = nil
+	require.Error(t, err)
+	require.Equal(t, expectedURL, u)
```

Comment:
here you can delete 
```go
var expectedURL *url.URL = nil
```
and use
```go
require.Nil(t, u)
```
instead.

Reply:
👍